pkg/controller/certificates/BUILD

```diff
@@ -10,10 +10,7 @@ go_library(
         "certificate_controller.go",
         "certificate_controller_utils.go",
     ],
-    visibility = [
```

Comment:
I don't understand this one?

Reply:
Does not seem like it's needed. verify bazel does not complain. So i just removed it.

Reply:
visibility is the bazel equivalent of import-restrictions.

Reply:
so, we shouldn't be removing these?